src/researchtikpy/get_videos_hashtag.py

```diff
@@ -163,11 +168,19 @@ def is_uninformative_backend_failure(response) -> bool:
     return response.status_code in status_codes and error_msg in err_msgs
 
 
-def error_message(response) -> str:
+def json_error_message(response) -> str:
     return response.json()["error"]["message"]
```

Comment:
I renamed this function to make it clear that it is extracting the error message from the `.json()` body.